include/ccl_constants.h

```diff
@@ -45,6 +45,7 @@ extern "C" {
  * Solar mass in units of kg (from GSL)
  */
 #define SOLAR_MASS GSL_CONST_MKSA_SOLAR_MASS
+//#define SOLAR_MASS 1.9885e30 //(from PDG 2015) in Kg
```

Comment:
This seems unrelated to the point of this PR, but it's fine to have it here in any case.